certified-connectors/Tilkee-V3/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.3",
+    "description": "Tilkee is an online platform that makes cross-functional collaboration happen. Combining deep knowledge of sales enablement with augmented intelligence, the vision of the team behind Tilkee is to align marketing with sales and guide Sales, Product Marketing, Channel Enablement, and Customer Success teams towards digital transformation.",
+    "title": "Tilkee V3",
+    "contact": {
+      "name": "Tilkee support",
+      "url": "https://help.tilkee.com",
+      "email": "support@tilkee.com"
+    }
+  },
+  "host": "endpoints.tilkee.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v3/oauth/authorize/": {},
+    "/v3/oauth/revoke/": {},
+    "/v3/oauth/token/": {},
+    "/v3/oauth/google/": {},
+    "/v3/oauth/google/callback/": {},
+    "/v3/oauth/linkedin/": {},
+    "/v3/oauth/linkedin/callback/": {},
+    "/v3/oauth/salesforce/": {},
+    "/v3/oauth/salesforce/callback/": {},
+    "/v3/oauth/microsoft/": {},
+    "/v3/oauth/microsoft/callback/": {},
+    "/v3/users/me/": {
+      "get": {
+        "description": "Retrieve current user details",
+        "summary": "Get current user information",
+        "tags": [
+          "Users"
+        ],
+        "operationId": "retrieve_me_v3_users_me__get",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "schema": {
+              "$ref": "#/definitions/User"
+            },
+            "headers": {}
+          }
+        },
+        "security": [
+          {
+            "oauth2_auth": []
```

Comment:
@FloxoS It is not required to have a "security" section for each operation. please remove it and test your connector thoroughly before pushing the updates to the PR.